omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -37,13 +37,14 @@
         <UpgradeVersion OnlyDetect='no'
                         Property='PREVIOUSFOUND'
                         Minimum='1.0.0' IncludeMinimum='yes'
-                        Maximum="99.99.99.99" IncludeMaximum='no'/>
+                        Maximum="$(var.VersionNumber)" IncludeMaximum='no'/>
```

Comment:
does this still allow same-version reinstalls? (I'd guess not since `IncludeMaximum` is set to `no`, but I may be wrong). Should we still allow same-version reinstalls? :)